indico/migrations/versions/20171214_1059_9c4418d7a6aa_add_session_types_table.py

```diff
@@ -0,0 +1,34 @@
+"""add session types table
+
+Revision ID: 9c4418d7a6aa
+Revises: 566d5de4e0e5
+Create Date: 2017-12-14 10:59:47.872426
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '9c4418d7a6aa'
+down_revision = '566d5de4e0e5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'session_types',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('event_id', sa.Integer(), nullable=False),
+        sa.Column('name', sa.String(), nullable=False),
+        sa.Column('is_poster', sa.Boolean(), nullable=False),
```

Comment:
what about `server_default` since it is not nullable?

Reply:
I thought we use `server_default` only when we add a new column to an existing table, so that the old records have some value.

Reply:
yeah, no need to do that when creating a new table

Reply:
ahh, sorry for that, didn't notice that it is indeed a new table ;)